radio/src/targets/common/arm/stm32/stm32_ws2812.h

```diff
@@ -25,7 +25,7 @@
 
 // RGB
 #define WS2812_BYTES_PER_LED 3
-#define WS2812_MAX_LEDS 8
+#define WS2812_MAX_LEDS 48
```

Comment:
Should this be set across the board, or be changable / configured for each target, as different targets will have different max counts?

Reply:
Or just use LED_STRIP_LENGTH instead and eliminate WS2812_MAX_LEDS?  Currently the driver only support one stance of WS2812.

Reply:
Ok, disregard this... I finally had another look at the code, and this change is correct... this was setting a cap on the maximum number of LEDS the driver will accept. As to whether this is actually needed or not.. that is a question for another day. :laughing: